### PR TITLE
Implement setting round duration

### PR DESCRIPTION
**Pull request by khooroko**
**Description**
Allows setting the duration from room settings

@lyhvictoria

**Impact**
- [x] Medium

**Risks**
Touched lobby scene

### CODE REVIEW

File: Assets/Scripts/NetworkController.cs [hunk 1]
```diff
@@ -294,24 +304,24 @@ private Toggle
         if (!PhotonNetwork.IsMasterClient) {
             if (changedProps.TryGetValue("RoomGoalType", out object temp)) {
                 if (temp is byte) {
-                    GoalType type = (GoalType)System.Enum.ToObject(typeof(GoalType) , temp);
+                    GoalType type = (GoalType)System.Enum.ToObject(typeof(GoalType), temp);
                     switch (type) {
                         case GoalType.REGULAR:
                             RegGoalToggle.isOn = true;
-                        break;
+                            break;
```
Comment (lyhvictoria) [inline Assets/Scripts/NetworkController.cs:312]: 👍

**Comment by lyhvictoria** [comment]
Oh, then is it possible to add round duration to [Playtest logging #83], so we will merge this first?

**Comment by khooroko** [comment]
ah ye ok

File: Assets/Scripts/GameManager.cs [hunk 2]
```diff
@@ -30,6 +31,20 @@ public class GameManager : MonoBehaviourPunCallbacks {
     }
 
     private void Start() {
+        if (PhotonNetwork.IsConnected) {
+            PhotonHashtable hash = PhotonNetwork.CurrentRoom.CustomProperties;
+            object temp;
+            if (hash.TryGetValue("RoomRoundDuration", out temp)) {
+                if (!(temp is byte)) {
```
Comment (lyhvictoria) [inline Assets/Scripts/GameManager.cs]: dang it

**Review by lyhvictoria — APPROVED** [review]

**Pull request merged**